ignite/templates/typed/map/stargate.go

```diff
@@ -237,17 +238,18 @@ func clientCliQueryModify(replacer placeholder.Replacer, opts *typed.Options) ge
 
 func genesisProtoModify(replacer placeholder.Replacer, opts *typed.Options) genny.RunFn {
 	return func(r *genny.Runner) error {
-		path := filepath.Join(opts.AppPath, "proto", opts.ModuleName, "genesis.proto")
+		path := filepath.Join(opts.AppPath, "proto", opts.AppName, opts.ModuleName, "genesis.proto")
 		f, err := r.Disk.Find(path)
 		if err != nil {
 			return err
 		}
 
-		templateProtoImport := `import "%[2]v/%[3]v.proto";
+		templateProtoImport := `import "%[2]v/%[3]v%[4]v.proto";
```

Comment:
```suggestion
		templateProtoImport := `import "%[2]v/%[3]v/%[4]v.proto";
```

Missing `/`

This is why CI is not happy :)

Reply:
Yep - and a bunch of other import path issues !

Reply:
These are the ones I could see that are still missing:

https://github.com/ignite/cli/blob/587e7e1e375b2918914d579f9505e8be45528164/ignite/templates/ibc/oracle.go#L185
https://github.com/ignite/cli/blob/587e7e1e375b2918914d579f9505e8be45528164/ignite/templates/ibc/packet.go#L215
https://github.com/ignite/cli/blob/587e7e1e375b2918914d579f9505e8be45528164/ignite/templates/ibc/packet.go#L299
https://github.com/ignite/cli/blob/587e7e1e375b2918914d579f9505e8be45528164/ignite/templates/message/stargate.go#L101
https://github.com/ignite/cli/blob/587e7e1e375b2918914d579f9505e8be45528164/ignite/templates/query/stargate.go#L89